frontend/app/model/Nav.scala

```diff
@@ -33,9 +33,9 @@ object Nav {
   )
 
   def footerNavigation(countryGroup: Option[CountryGroup] = None) = List(
-    NavItem("help", routes.Info.help().toString, "Help"),
```

Comment:
Do we not need the help link any more?

Reply:
The old help link contained the MC FAQ and Live FAQs, the membership FAQ is now deleted by Charles. 

Basically FAQs = the old help link.